civihr_default_theme/scss/pages/_onboarding-wizard.scss

```diff
@@ -1,4 +1,5 @@
 $carousel-box-shadow-color: rgba(#3f3c55,0.3);
+$crm-darker-grayblue: rgb(182, 186, 207);
```

Comment:
in shoreditch we have now `$crm-grayblue-darker`, why do we need to use `$crm-darker-grayblue` in here instead?

Reply:
There is no need of this variable, will remove it. thanks for catching it.